### DISCUSSION TITLE
Repurpose the Combat Spacer in the Main tab

### DISCUSSION THREAD

**Discussion by Re4XN** [fh-fvtt/zweihander#73] [Ideas]
While the Combat Spacer element looks really cool, I feel like the space could be repurposed to provide more functionality to the users. An example, illustrated below, could be to implement a generic way of rolling the most common set of dice:

![image](https://user-images.githubusercontent.com/16267371/154360309-dc62f25f-507c-4119-97cf-56d509eb18a1.png)

**Comment by ghost** [comment]
Yeah, while I like the spacers visual appeal (I need to like it for the fact that I invested hours into that animation :D) I wouldn't mind giving it up for good functionality. Would be even better if that functionality could make good use of the extra space when the sheet is resized.
What sets of dice do you have in mind exactly? I would like the feature to be more specific and shiny than generic dice sets (for which there are already great foundry modules out there) but if we fail to find something else then that's a good starting point!

**Reply by Re4XN** [reply]
> I would like the feature to be more specific and shiny than generic dice sets

Yeah, it was just an idea for a starting point. I think Zweihänder uses d6, d10 and d100 _only_ (but I am not sure), so generic rolls might not be the best use of that space. A couple more specific ideas:

* Buttons to roll Stress (1d10+1), Terror (2d10+2) and Horror (3d10+3), which would apply the result automatically to the character's Peril Condition Track.
* A button to roll Fury dice.
* A button to roll Chaos dice.

Of course, this is a suggestion. If you have other ideas to fill that space with, I'm all ears!
